.github/workflows/daily_common.yml

```diff
@@ -96,3 +96,4 @@ jobs:
 
           export NIMFLAGS="${NIMFLAGS} --mm:${{ matrix.nim.memory_management }}"
           nimble test
+          nimble testautotls
```

Comment:
we need to separate `testautotls` in part that tests `api` and anything else that may come later.
could this test be `testautotels_api`?

Reply:
or `testautotels_integration`

Reply:
Disagree. All AutoTLS tests are integration tests. All AutoTLS does is integrate (with acme, and with broker).

Reply:
> AutoTLS tests are integration tests.

super! lets name them in this spirit.